### PR TITLE
Add VRAP logo and info

### PR DESCRIPTION
**Pull request by sujithsomraaj**

### CODE REVIEW

**Comment by catenocrypt** [comment]
Correction needed:  the folder name corresponding to the token contract should be in checksum format.  Please rename (or create new PR with correct version, no need to pay again).

```
 -  XXX 'Folder structure for chain smartchain (ethereum fork)':  1 errors
   XXX   'Expect asset at path /home/runner/work/assets/assets/blockchains/smartchain/assets/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e in checksum: '0x271C418B045d05A1D52c6bF849d47b5B5B4d769e''
```

**Comment by sujithsomraaj** [comment]
Hi, I've created a new PR with ID #5906 Please Check Again. Thanks

File: blockchains/smartchain/assets/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e/info.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            "url": "https://t.me/veraswap",
+            "handle": "Veraswap Protocol"
+        }
+    ],
+    "explorer": "https://bscscan.com/token/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e",
+    "type": "BEP20",
+    "symbol": "VRAP",
+    "decimals": 18,
+    "status": "active",
+    "id": "0x271c418b045d05a1d52c6bf849d47b5b5b4d769e"
```
Comment (vikmeup) [inline blockchains/smartchain/assets/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e/info.json:23]: `0x271c418b045d05a1d52c6bf849d47b5b5b4d769e` should be checksummed version for info and file name: `0x271C418B045d05A1D52c6bF849d47b5B5B4d769e`

Comment (sujithsomraaj) [reply blockchains/smartchain/assets/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e/info.json:23]: https://github.com/trustwallet/assets/pull/5906

I've created a new pull request with the correct info as I cannot edit the current request

Comment (sujithsomraaj) [reply blockchains/smartchain/assets/0x271c418b045d05a1d52c6bf849d47b5b5b4d769e/info.json:23]: Pleae check the above commit

**Pull request closed without merging**